### PR TITLE
Create Redis as Layer1 service

### PR DESCRIPTION
**Pull request by tlake**

### CODE REVIEW

File: module/main.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
+resource "layer0_load_balancer" "redis" {
+  name = "${var.load_balancer_name}"
+  environment = "${var.environment_id}"
+  private = true
```
Comment (zpatrick) [inline module/main.tf]: let's make `private` configurable

File: module/main.tf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  port {
+    host_port = "${var.redis_port}"
```
Comment (zpatrick) [inline module/main.tf]: [nit] `redis_port` -> `port`

File: module/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+output "load_balancer_url" {
```
Comment (zpatrick) [inline module/outputs.tf]: Let's add the ids of all created entities, e.g. https://github.com/quintilesims/consul/blob/master/terraform/outputs.tf

File: README.md [hunk 4]
```diff
@@ -1,2 +1,9 @@
-# redis
-Official Layer0 Redis Service
+# Layer0 Redis
```
Comment (zpatrick) [inline README.md:1]: Can we make this (and all future) layer1 repos match https://github.com/quintilesims/consul?

```
/
README.md
example/
terraform/
```

File: module/main.tf [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    container_port = "${var.redis_port}"
+    protocol = "tcp"
+  }
+}
+
+resource "layer0_service" "redis" {
+  name = "${var.service_name}"
```
Comment (zpatrick) [inline module/main.tf]: make sure to run `terraform fmt` on these files

**Review by zpatrick — CHANGES_REQUESTED** [review]

File: module/main.tf (hunk 5 above)
Comment (tlake) [reply module/main.tf]: TIL

File: module/main.tf (hunk 1 above)
Comment (tlake) [reply module/main.tf]: ✅

File: module/main.tf (hunk 2 above)
Comment (tlake) [reply module/main.tf]: ✅

File: README.md (hunk 4 above)
Comment (tlake) [reply README.md:1]: ✅

File: module/main.tf (hunk 5 above)
Comment (tlake) [reply module/main.tf]: ✅ 

_and_ I have a new Vim plugin

File: module/outputs.tf (hunk 3 above)
Comment (tlake) [reply module/outputs.tf]: ✅

**Comment by zpatrick** [comment]
:+1: looks good to me!

**Pull request merged**